kubo-rpc/src/lib.rs

```diff
@@ -245,6 +253,31 @@ impl IpfsDep for Arc<IpfsService> {
             .await
             .map_err(Error::Internal)?)
     }
+    async fn import<T: std::io::Read + std::io::Seek + Send>(
+        &self,
+        car: T,
+    ) -> Result<Vec<Cid>, Error> {
+        let mut reader =
+            blockless_car::reader::new_v1(car).map_err(|e| Error::Internal(e.into()))?;
+        let roots = reader.header().roots();
+        let mut stored = vec![];
+        for cid in roots.iter() {
```

Comment:
I don't believe all roots will be in the `roots`. Normally only one of the CIDs are there. Maybe try iterating through `reader.sections()`?

Reply:
Do we care about the roots at all?
I thought that the import api took all the blocks and stored them wether they were in the IPLD tree from the root or not.

Reply:
We don't have to care about which CID is root no. We should just import all of the blocks.

Reply:
Iteration is now be over the blocks.

For roots, we're aiming for parity with the Go implementation as per the IPLD [CARv1 spec](https://ipld.io/specs/transport/car/carv1/#number-of-roots). We'll error out if there is no root and return the first root if there is more than one and not throw an error in that case.

Reply:
Great!